src/plugins/intel_cpu/tests/functional/single_layer_tests/softmax.cpp

```diff
@@ -64,7 +64,7 @@ class SoftMaxLayerCPUTest : public testing::WithParamInterface<softmaxCPUTestPar
         }
 
         if (inType == ElementType::bf16) {
-            rel_threshold = 1e-2f;
+            rel_threshold = 2e-2f;
```

Comment:
after "ref" was removed from the skip pattern softmax bf16 tests started to execute
and 0.01 threshold is actually not enough